tdlib 1.7.0 linker command failed in aarch64-darwin

### Describe the bug
```console
ld: file not found: /nix/store/vf55lp3k5xgx1vlkysb7pbw76f1wh21l-clang-wrapper-11.1.0/resource-root/lib/darwin/libclang_rt.ubsan_osx_dynamic.dylib
```
### Steps To Reproduce
```console
nix-build '<nixpkgs>' -A tdlib
```
### Expected behavior
```console
bzcat /nix/var/log/nix/drvs/iv/78wkcqsxd36zn2809d4dbn5xjsyz1y-tdlib-1.7.0.drv.bz2
```
```console
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/crni5bw76gxd8z4nmxa27ywscjjd6mia-source
source root is source
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0/share/doc/TDLib -DCMAKE_INSTALL_INFODIR=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0/include -DCMAKE_INSTALL_SBINDIR=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/vf55lp3k5xgx1vlkysb7pbw76f1wh21l-clang-wrapper-11.1.0/bin/strip -DCMAKE_RANLIB=/nix/store/mp5xsfnvdbdyqjmcipyks4nph534jc3q-cctools-binutils-darwin-949.0.1/bin/ranlib -DCMAKE_AR=/nix/store/mp5xsfnvdbdyqjmcipyks4nph534jc3q-cctools-binutils-darwin-949.0.1/bin/ar -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=/nix/store/ip9nwynps1bkzx4x4835p7n5nx3vrqmi-tdlib-1.7.0
-- The CXX compiler identification is Clang 11.1.0
-- The C compiler identification is Clang 11.1.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/vf55lp3k5xgx1vlkysb7pbw76f1wh21l-clang-wrapper-11.1.0/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/vf55lp3k5xgx1vlkysb7pbw76f1wh21l-clang-wrapper-11.1.0/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Could NOT find ccache (this is NOT an error)
-- Found OpenSSL: /nix/store/vd5p1q0yhnqm5znqhcj97s9a2jz7bf3q-openssl-1.1.1l/lib/libcrypto.dylib (found version "1.1.1l")
-- Found OpenSSL: /nix/store/mvl7h82cc57d2knzm0372z0dq4g8gdqj-openssl-1.1.1l-dev/include /nix/store/vd5p1q0yhnqm5znqhcj97s9a2jz7bf3q-openssl-1.1.1l/lib/libssl.dylib;/nix/store/vd5p1q0yhnqm5znqhcj97s9a2jz7bf3q-openssl-1.1.1l/lib/libcrypto.dylib
-- Looking for pthread.h
-- Looking for pthread.h - found
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Performing Test HAVE_STD14
-- Performing Test HAVE_STD14 - Success
-- Performing Test HAVE_CXX_FLAG_WALL
-- Performing Test HAVE_CXX_FLAG_WALL - Success
-- Performing Test HAVE_CXX_FLAG_WEXTRA
-- Performing Test HAVE_CXX_FLAG_WEXTRA - Success
-- Performing Test HAVE_CXX_FLAG_WIMPLICIT_FALLTHROUGH_2
-- Performing Test HAVE_CXX_FLAG_WIMPLICIT_FALLTHROUGH_2 - Failed
-- Performing Test HAVE_CXX_FLAG_WPOINTER_ARITH
-- Performing Test HAVE_CXX_FLAG_WPOINTER_ARITH - Success
-- Performing Test HAVE_CXX_FLAG_WCAST_QUAL
-- Performing Test HAVE_CXX_FLAG_WCAST_QUAL - Success
-- Performing Test HAVE_CXX_FLAG_WSIGN_COMPARE
-- Performing Test HAVE_CXX_FLAG_WSIGN_COMPARE - Success
-- Performing Test HAVE_CXX_FLAG_WDUPLICATED_BRANCHES
-- Performing Test HAVE_CXX_FLAG_WDUPLICATED_BRANCHES - Failed
-- Performing Test HAVE_CXX_FLAG_WDUPLICATED_COND
-- Performing Test HAVE_CXX_FLAG_WDUPLICATED_COND - Failed
-- Performing Test HAVE_CXX_FLAG_WALLOC_ZERO
-- Performing Test HAVE_CXX_FLAG_WALLOC_ZERO - Failed
-- Performing Test HAVE_CXX_FLAG_WLOGICAL_OP
-- Performing Test HAVE_CXX_FLAG_WLOGICAL_OP - Failed
-- Performing Test HAVE_CXX_FLAG_WTAUTOLOGICAL_COMPARE
-- Performing Test HAVE_CXX_FLAG_WTAUTOLOGICAL_COMPARE - Success
-- Performing Test HAVE_CXX_FLAG_WVLA
-- Performing Test HAVE_CXX_FLAG_WVLA - Success
-- Performing Test HAVE_CXX_FLAG_WNON_VIRTUAL_DTOR
-- Performing Test HAVE_CXX_FLAG_WNON_VIRTUAL_DTOR - Success
-- Performing Test HAVE_CXX_FLAG_WUNUSED_PARAMETER
-- Performing Test HAVE_CXX_FLAG_WUNUSED_PARAMETER - Success
-- Performing Test HAVE_CXX_FLAG_WCONVERSION
-- Performing Test HAVE_CXX_FLAG_WCONVERSION - Success
-- Performing Test HAVE_CXX_FLAG_WSIGN_CONVERSION
-- Performing Test HAVE_CXX_FLAG_WSIGN_CONVERSION - Success
-- Performing Test HAVE_CXX_FLAG_WCXX14_COMPAT_PEDANTIC
-- Performing Test HAVE_CXX_FLAG_WCXX14_COMPAT_PEDANTIC - Success
-- Performing Test HAVE_CXX_FLAG_WDEPRECATED
-- Performing Test HAVE_CXX_FLAG_WDEPRECATED - Success
-- Performing Test HAVE_CXX_FLAG_WUNUSED_COMMAND_LINE_ARGUMENT
-- Performing Test HAVE_CXX_FLAG_WUNUSED_COMMAND_LINE_ARGUMENT - Success
-- Performing Test HAVE_CXX_FLAG_QUNUSED_ARGUMENTS
-- Performing Test HAVE_CXX_FLAG_QUNUSED_ARGUMENTS - Success
-- Performing Test HAVE_CXX_FLAG_WODR
-- Performing Test HAVE_CXX_FLAG_WODR - Success
-- Performing Test HAVE_CXX_FLAG_FLTO_ODR_TYPE_MERGING
-- Performing Test HAVE_CXX_FLAG_FLTO_ODR_TYPE_MERGING - Failed
-- Found ZLIB: /nix/store/p7j966bh29ffsj4vvia7mis6iq3y9vlv-zlib-1.2.11/lib/libz.dylib (found version "1.2.11")
-- Found ZLIB: /nix/store/lcw5w5hfza00768i7x8lv18sgjvp6nzy-zlib-1.2.11-dev/include /nix/store/p7j966bh29ffsj4vvia7mis6iq3y9vlv-zlib-1.2.11/lib/libz.dylib
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
-- Performing Test GNU_READLINE_FOUND
-- Performing Test GNU_READLINE_FOUND - Success
-- Found Readline: /nix/store/prmyj26g2f8c2vpjrad7p9n6yma3f1hq-readline-6.3p08-dev/include
-- Found Readline: /nix/store/prmyj26g2f8c2vpjrad7p9n6yma3f1hq-readline-6.3p08-dev/include /nix/store/si3vbhy05wjwnpgjpzrl1c5dbwja5kwq-readline-6.3p08/lib/libreadline.dylib
-- Performing Test USABLE_READLINE_FOUND
-- Performing Test USABLE_READLINE_FOUND - Success
-- Configuring done
CMake Warning (dev):
  Policy CMP0068 is not set: RPATH settings on macOS do not affect
  install_name.  Run "cmake --help-policy CMP0068" for policy details.  Use
  the cmake_policy command to set the policy and suppress this warning.

  For compatibility with older versions of CMake, the install_name fields for
  the following targets are still affected by RPATH settings:

   tdjson

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done
CMake Warning:
  Manually-specified variables were not used by the project:

    BUILD_TESTING
    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_INSTALL_DOCDIR
    CMAKE_INSTALL_INFODIR
    CMAKE_INSTALL_LIBEXECDIR
    CMAKE_INSTALL_LOCALEDIR
    CMAKE_INSTALL_MANDIR
    CMAKE_INSTALL_OLDINCLUDEDIR
    CMAKE_INSTALL_SBINDIR
    CMAKE_POLICY_DEFAULT_CMP0025


-- Build files have been written to: /tmp/nix-build-tdlib-1.7.0.drv-0/source/build
cmake: enabled parallel building
@nix { "action": "setPhase", "phase": "buildPhase" }
building
build flags: -j8 -l8 SHELL=/nix/store/bkp2nl65nbzjkvg2nypxxdjns7c13g8p-bash-5.1-p8/bin/bash
[  0%] Building CXX object tdtl/CMakeFiles/tdtl.dir/td/tl/tl_core.cpp.o
[  0%] Building C object sqlite/CMakeFiles/tdsqlite.dir/sqlite/sqlite3.c.o
[  0%] Building CXX object tdutils/generate/CMakeFiles/generate_mime_types_gperf.dir/generate_mime_types_gperf.cpp.o
[  0%] Building CXX object tdtl/CMakeFiles/tdtl.dir/td/tl/tl_config.cpp.o
[  0%] Generate git_commit.h
[  1%] Building CXX object tdtl/CMakeFiles/tdtl.dir/td/tl/tl_file_outputer.cpp.o
[  1%] Building CXX object tdtl/CMakeFiles/tdtl.dir/td/tl/tl_file_utils.cpp.o
[  1%] Building CXX object tdtl/CMakeFiles/tdtl.dir/td/tl/tl_generate.cpp.o
[  1%] Building CXX object tdtl/CMakeFiles/tdtl.dir/td/tl/tl_outputer.cpp.o
./gen_git_commit_h.sh: line 2: git: command not found
./gen_git_commit_h.sh: line 3: git: command not found
[  1%] Building CXX object tdtl/CMakeFiles/tdtl.dir/td/tl/tl_string_outputer.cpp.o
[  2%] Building CXX object tdtl/CMakeFiles/tdtl.dir/td/tl/tl_writer.cpp.o
[  2%] Built target git_commit
[  3%] Linking CXX executable generate_mime_types_gperf
[  3%] Linking CXX static library libtdtl.a
[  3%] Built target tdtl
[  3%] Built target generate_mime_types_gperf
[  4%] Building CXX object td/generate/CMakeFiles/remove_documentation.dir/remove_documentation.cpp.o
[  4%] Building CXX object td/generate/CMakeFiles/generate_c.dir/generate_c.cpp.o
[  4%] Building CXX object td/generate/CMakeFiles/td_generate_java_api.dir/tl_writer_java.cpp.o
[  4%] Building CXX object td/generate/CMakeFiles/generate_common.dir/generate_common.cpp.o
[  4%] Building CXX object td/generate/CMakeFiles/generate_common.dir/tl_writer_cpp.cpp.o
[  5%] Building CXX object td/generate/CMakeFiles/td_generate_java_api.dir/generate_java.cpp.o
[  5%] Building CXX object td/generate/CMakeFiles/generate_common.dir/tl_writer_h.cpp.o
[  5%] Linking CXX executable remove_documentation
[  5%] Building CXX object td/generate/CMakeFiles/generate_common.dir/tl_writer_hpp.cpp.o
[  5%] Linking CXX executable td_generate_java_api
[  5%] Built target remove_documentation
[  6%] Building CXX object td/generate/CMakeFiles/generate_common.dir/tl_writer_jni_cpp.cpp.o
[  6%] Building CXX object td/generate/CMakeFiles/generate_common.dir/tl_writer_jni_h.cpp.o
[  6%] Generating ../../../tdutils/generate/auto/mime_type_to_extension.gperf, ../../../tdutils/generate/auto/extension_to_mime_type.gperf
[  6%] Building CXX object td/generate/CMakeFiles/generate_common.dir/tl_writer_td.cpp.o
[  6%] Built target td_generate_java_api
Extension "wmz" matches more than one type
Extension "sub" matches more than one type
[  6%] Generating ../../../tdutils/generate/auto/mime_type_to_extension.cpp
[  6%] Generating ../../../tdutils/generate/auto/extension_to_mime_type.cpp
[  7%] Linking CXX executable generate_c
[  7%] Linking CXX executable generate_common
[  7%] Built target generate_c
[  7%] Built target generate_common
[  7%] Generate common tl source files
Write tl to file auto/td/telegram/telegram_api.cpp
Write tl to file auto/td/telegram/telegram_api.h
Write tl to file auto/td/telegram/telegram_api.hpp
Write tl to file auto/td/telegram/secret_api.cpp
Write tl to file auto/td/telegram/secret_api.h
Write tl to file auto/td/telegram/secret_api.hpp
Write tl to file auto/td/mtproto/mtproto_api.cpp
Write tl to file auto/td/mtproto/mtproto_api.h
Write tl to file auto/td/mtproto/mtproto_api.hpp
Write tl to file auto/td/telegram/td_api.cpp
Write tl to file auto/td/telegram/td_api.h
Write tl to file auto/td/telegram/td_api.hpp
[  7%] Built target tl_generate_common
[  7%] Built target tdmime_auto
[  7%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/FileFd.cpp.o
[  7%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/PollFlags.cpp.o
[  7%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/path.cpp.o
[  7%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/rlimit.cpp.o
[  7%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/MemoryMapping.cpp.o
[  8%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/IPAddress.cpp.o
[  8%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/Clocks.cpp.o
[  9%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/ServerSocketFd.cpp.o
[  9%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/signals.cpp.o
[  9%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/sleep.cpp.o
[  9%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/SocketFd.cpp.o
[ 10%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/stacktrace.cpp.o
[ 10%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/Stat.cpp.o
[ 10%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/StdStreams.cpp.o
[ 10%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/thread_local.cpp.o
[ 12%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/UdpSocketFd.cpp.o
[ 12%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/uname.cpp.o
[ 12%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/user.cpp.o
[ 12%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/wstring_convert.cpp.o
[ 13%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/Epoll.cpp.o
[ 13%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/EventFdBsd.cpp.o
[ 13%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/EventFdLinux.cpp.o
[ 13%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/EventFdWindows.cpp.o
[ 13%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/Iocp.cpp.o
[ 14%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/KQueue.cpp.o
[ 14%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/NativeFd.cpp.o
[ 14%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/Poll.cpp.o
[ 14%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/Select.cpp.o
[ 15%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/ThreadIdGuard.cpp.o
[ 15%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/ThreadPthread.cpp.o
[ 15%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/port/detail/WineventPoll.cpp.o
[ 15%] Building CXX object tdutils/CMakeFiles/tdutils.dir/generate/auto/mime_type_to_extension.cpp.o
[ 16%] Building CXX object tdutils/CMakeFiles/tdutils.dir/generate/auto/extension_to_mime_type.cpp.o
[ 16%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/base64.cpp.o
[ 16%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/BigNum.cpp.o
[ 16%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/buffer.cpp.o
[ 17%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/BufferedUdp.cpp.o
[ 17%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/check.cpp.o
[ 17%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/crypto.cpp.o
[ 17%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/ExitGuard.cpp.o
[ 18%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/FileLog.cpp.o
[ 18%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/find_boundary.cpp.o
[ 18%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/filesystem.cpp.o
[ 18%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/Gzip.cpp.o
[ 18%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/GzipByteFlow.cpp.o
[ 19%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/Hints.cpp.o
[ 19%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/HttpUrl.cpp.o
[ 19%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/JsonBuilder.cpp.o
[ 19%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/logging.cpp.o
[ 20%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/misc.cpp.o
[ 20%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/MpmcQueue.cpp.o
[ 20%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/OptionParser.cpp.o
[ 20%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/PathView.cpp.o
[ 21%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/Random.cpp.o
[ 21%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/SharedSlice.cpp.o
[ 21%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/Slice.cpp.o
[ 21%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/StackAllocator.cpp.o
[ 23%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/Status.cpp.o
[ 23%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/StringBuilder.cpp.o
[ 23%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/tests.cpp.o
[ 23%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/Time.cpp.o
[ 24%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/Timer.cpp.o
[ 24%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/TsFileLog.cpp.o
[ 24%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/tl_parsers.cpp.o
[ 24%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/translit.cpp.o
[ 24%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/unicode.cpp.o
[ 25%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/utf8.cpp.o
[ 25%] Building CXX object tdutils/CMakeFiles/tdutils.dir/td/utils/MimeType.cpp.o
[ 25%] Linking CXX static library libtdutils.a
[ 25%] Built target tdutils
[ 25%] Building CXX object tdactor/CMakeFiles/tdactor.dir/td/actor/ConcurrentScheduler.cpp.o
[ 25%] Building CXX object CMakeFiles/memprof.dir/memprof/memprof.cpp.o
[ 26%] Building CXX object td/generate/CMakeFiles/generate_json.dir/generate_json.cpp.o
[ 26%] Building CXX object benchmark/CMakeFiles/check_tls.dir/check_tls.cpp.o
[ 27%] Building CXX object CMakeFiles/tdapi.dir/td/generate/auto/td/telegram/td_api.cpp.o
[ 27%] Building CXX object benchmark/CMakeFiles/bench_queue.dir/bench_queue.cpp.o
[ 27%] Building CXX object benchmark/CMakeFiles/bench_log.dir/bench_log.cpp.o
[ 27%] Linking CXX static library libmemprof.a
[ 27%] Built target memprof
[ 27%] Building CXX object td/generate/CMakeFiles/generate_json.dir/tl_json_converter.cpp.o
[ 27%] Building CXX object benchmark/CMakeFiles/rmdir.dir/rmdir.cpp.o
[ 28%] Linking CXX executable bench_log
[ 29%] Linking CXX executable bench_queue
[ 29%] Built target bench_log
[ 30%] Linking CXX executable check_tls
[ 30%] Building CXX object benchmark/CMakeFiles/bench_crypto.dir/bench_crypto.cpp.o
[ 30%] Linking CXX executable rmdir
[ 30%] Built target bench_queue
[ 31%] Building CXX object tdactor/CMakeFiles/tdactor.dir/td/actor/impl/Scheduler.cpp.o
[ 31%] Building CXX object benchmark/CMakeFiles/bench_empty.dir/bench_empty.cpp.o
[ 31%] Built target check_tls
[ 31%] Building CXX object tdactor/CMakeFiles/tdactor.dir/td/actor/MultiPromise.cpp.o
[ 31%] Built target rmdir
[ 31%] Building CXX object tdactor/CMakeFiles/tdactor.dir/td/actor/Timeout.cpp.o
[ 31%] Linking CXX executable bench_empty
[ 31%] Built target bench_empty
[ 31%] Linking CXX executable bench_crypto
[ 31%] Linking CXX executable generate_json
[ 31%] Built target bench_crypto
[ 31%] Built target generate_json
[ 31%] Generate JSON tl source files
[ 31%] Linking CXX static library libtdactor.a
[ 31%] Built target tdactor
[ 31%] Building CXX object tdactor/CMakeFiles/example.dir/example/example.cpp.o
[ 31%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/HttpChunkedByteFlow.cpp.o
[ 31%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/HttpConnectionBase.cpp.o
[ 32%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/GetHostByNameActor.cpp.o
[ 32%] Building CXX object benchmark/CMakeFiles/bench_actor.dir/bench_actor.cpp.o
[ 32%] Built target tl_generate_json
[ 32%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/HttpContentLengthByteFlow.cpp.o
[ 34%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/HttpFile.cpp.o
[ 34%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/HttpInboundConnection.cpp.o
[ 34%] Linking CXX executable example
[ 34%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/HttpOutboundConnection.cpp.o
[ 34%] Built target example
[ 34%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/HttpProxy.cpp.o
[ 35%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/HttpQuery.cpp.o
[ 35%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/HttpReader.cpp.o
[ 35%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/Socks5.cpp.o
[ 35%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/SslStream.cpp.o
[ 35%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/TcpListener.cpp.o
[ 36%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/TransparentProxy.cpp.o
[ 36%] Building CXX object tdnet/CMakeFiles/tdnet.dir/td/net/Wget.cpp.o
[ 36%] Linking CXX executable bench_actor
[ 36%] Built target bench_actor
[ 36%] Linking CXX static library libtdnet.a
[ 36%] Built target tdnet
[ 36%] Building CXX object benchmark/CMakeFiles/bench_http_server_fast.dir/bench_http_server_fast.cpp.o
[ 36%] Building CXX object benchmark/CMakeFiles/wget.dir/wget.cpp.o
[ 36%] Building CXX object benchmark/CMakeFiles/bench_http_server.dir/bench_http_server.cpp.o
[ 37%] Building CXX object benchmark/CMakeFiles/bench_http_reader.dir/bench_http_reader.cpp.o
[ 38%] Building CXX object benchmark/CMakeFiles/bench_http_server_cheat.dir/bench_http_server_cheat.cpp.o
[ 38%] Building CXX object benchmark/CMakeFiles/bench_http.dir/bench_http.cpp.o
[ 39%] Linking C static library libtdsqlite.a
[ 39%] Built target tdsqlite
[ 40%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/binlog/Binlog.cpp.o
[ 40%] Linking CXX executable bench_http_reader
[ 41%] Linking CXX executable wget
[ 41%] Built target bench_http_reader
[ 41%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/binlog/BinlogEvent.cpp.o
[ 41%] Linking CXX executable bench_http
[ 41%] Linking CXX executable bench_http_server_cheat
[ 41%] Built target wget
[ 41%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/binlog/ConcurrentBinlog.cpp.o
[ 41%] Linking CXX executable bench_http_server
[ 41%] Built target bench_http_server_cheat
[ 41%] Built target bench_http
[ 41%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/binlog/detail/BinlogEventsBuffer.cpp.o
[ 42%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/binlog/detail/BinlogEventsProcessor.cpp.o
[ 42%] Linking CXX executable bench_http_server_fast
[ 42%] Built target bench_http_server
[ 42%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/SqliteConnectionSafe.cpp.o
[ 42%] Built target bench_http_server_fast
[ 42%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/SqliteDb.cpp.o
[ 42%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/SqliteKeyValue.cpp.o
[ 43%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/SqliteKeyValueAsync.cpp.o
[ 43%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/SqliteStatement.cpp.o
[ 43%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/TQueue.cpp.o
[ 43%] Building CXX object tddb/CMakeFiles/tddb.dir/td/db/detail/RawSqliteDb.cpp.o
[ 45%] Linking CXX static library libtddb.a
[ 45%] Built target tddb
[ 46%] Building CXX object tddb/CMakeFiles/binlog_dump.dir/td/db/binlog/binlog_dump.cpp.o
[ 46%] Building CXX object benchmark/CMakeFiles/bench_db.dir/bench_db.cpp.o
[ 47%] Linking CXX executable binlog_dump
[ 47%] Built target binlog_dump
[ 47%] Linking CXX static library libtdapi.a
[ 47%] Built target tdapi
[ 47%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/DhHandshake.cpp.o
[ 47%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/KDF.cpp.o
[ 48%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/AuthData.cpp.o
[ 48%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/Handshake.cpp.o
[ 48%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/HandshakeActor.cpp.o
[ 48%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/HttpTransport.cpp.o
[ 49%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/IStreamTransport.cpp.o
[ 49%] Linking CXX executable bench_db
[ 49%] Built target bench_db
[ 49%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/Ping.cpp.o
[ 49%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/PingConnection.cpp.o
[ 50%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/ProxySecret.cpp.o
[ 50%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/RawConnection.cpp.o
[ 50%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/RSA.cpp.o
[ 50%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/SessionConnection.cpp.o
[ 51%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/TcpTransport.cpp.o
[ 51%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/TlsInit.cpp.o
[ 51%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/TlsReaderByteFlow.cpp.o
[ 51%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/Transport.cpp.o
[ 52%] Building CXX object CMakeFiles/tdcore.dir/td/mtproto/utils.cpp.o
[ 52%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/AnimationsManager.cpp.o
[ 52%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/AudiosManager.cpp.o
[ 52%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/AuthManager.cpp.o
[ 53%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/AutoDownloadSettings.cpp.o
[ 53%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/BackgroundManager.cpp.o
[ 53%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/BackgroundType.cpp.o
[ 53%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/CallActor.cpp.o
[ 53%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/CallDiscardReason.cpp.o
[ 54%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/CallManager.cpp.o
[ 54%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/CallbackQueriesManager.cpp.o
[ 54%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/ClientActor.cpp.o
[ 54%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/ConfigManager.cpp.o
[ 54%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/Contact.cpp.o
[ 56%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/ConfigShared.cpp.o
[ 56%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/ContactsManager.cpp.o
[ 56%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/CountryInfoManager.cpp.o
[ 57%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DelayDispatcher.cpp.o
[ 57%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/Dependencies.cpp.o
[ 57%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DeviceTokenManager.cpp.o
[ 57%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DhCache.cpp.o
[ 58%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DialogAction.cpp.o
[ 58%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DialogAdministrator.cpp.o
[ 58%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DialogDb.cpp.o
[ 58%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DialogFilter.cpp.o
[ 59%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DialogId.cpp.o
[ 59%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DialogLocation.cpp.o
[ 59%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DialogParticipant.cpp.o
[ 59%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DialogSource.cpp.o
[ 59%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/Document.cpp.o
[ 60%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DocumentsManager.cpp.o
[ 60%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/DraftMessage.cpp.o
[ 60%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileBitmask.cpp.o
[ 60%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/FileReferenceManager.cpp.o
[ 61%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileDb.cpp.o
[ 61%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileDownloader.cpp.o
[ 61%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileEncryptionKey.cpp.o
[ 61%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileFromBytes.cpp.o
[ 62%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileGcParameters.cpp.o
[ 62%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileGcWorker.cpp.o
[ 62%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileGenerateManager.cpp.o
[ 62%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileHashUploader.cpp.o
[ 63%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileLoader.cpp.o
[ 63%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileLoaderUtils.cpp.o
[ 63%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileLoadManager.cpp.o
[ 63%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileManager.cpp.o
[ 63%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileStats.cpp.o
[ 64%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileStatsWorker.cpp.o
[ 64%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileType.cpp.o
[ 64%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/FileUploader.cpp.o
[ 64%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/PartsManager.cpp.o
[ 65%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/files/ResourceManager.cpp.o
[ 65%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/Game.cpp.o
[ 65%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/Global.cpp.o
[ 65%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/HashtagHints.cpp.o
[ 67%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/InlineQueriesManager.cpp.o
[ 67%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/InputDialogId.cpp.o
[ 67%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/InputMessageText.cpp.o
[ 67%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/JsonValue.cpp.o
[ 68%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/LanguagePackManager.cpp.o
[ 68%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/Location.cpp.o
[ 68%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/logevent/LogEventHelper.cpp.o
[ 68%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/Logging.cpp.o
[ 69%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/MessageContent.cpp.o
[ 69%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/MessageContentType.cpp.o
[ 69%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/MessageEntity.cpp.o
[ 69%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/MessageId.cpp.o
[ 69%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/MessageReplyInfo.cpp.o
[ 70%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/MessagesDb.cpp.o
[ 70%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/MessageSearchFilter.cpp.o
[ 70%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/MessagesManager.cpp.o
[ 70%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/misc.cpp.o
[ 71%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/AuthDataShared.cpp.o
[ 71%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/ConnectionCreator.cpp.o
[ 71%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/DcAuthManager.cpp.o
[ 71%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/DcOptionsSet.cpp.o
[ 72%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/MtprotoHeader.cpp.o
[ 72%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/NetActor.cpp.o
[ 72%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/NetQuery.cpp.o
[ 72%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/NetQueryCreator.cpp.o
[ 73%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/NetQueryDelayer.cpp.o
[ 73%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/NetQueryDispatcher.cpp.o
[ 73%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/NetQueryStats.cpp.o
[ 73%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/NetStatsManager.cpp.o
[ 74%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/Proxy.cpp.o
[ 74%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/PublicRsaKeyShared.cpp.o
[ 74%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/PublicRsaKeyWatchdog.cpp.o
[ 74%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/Session.cpp.o
[ 74%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/SessionProxy.cpp.o
[ 75%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/net/SessionMultiProxy.cpp.o
[ 75%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/NotificationManager.cpp.o
[ 75%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/NotificationSettings.cpp.o
[ 75%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/NotificationType.cpp.o
[ 76%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/Payments.cpp.o
[ 76%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/PasswordManager.cpp.o
[ 76%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/PhoneNumberManager.cpp.o
[ 76%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/PrivacyManager.cpp.o
[ 78%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/Photo.cpp.o
[ 78%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/PhotoSizeSource.cpp.o
[ 78%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/PollManager.cpp.o
[ 78%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/QueryCombiner.cpp.o
[ 79%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/ReplyMarkup.cpp.o
[ 79%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/RestrictionReason.cpp.o
[ 79%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/SecretChatActor.cpp.o
[ 79%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/SecretChatDb.cpp.o
[ 80%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/SecretChatsManager.cpp.o
[ 80%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/SecureManager.cpp.o
[ 80%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/SecureStorage.cpp.o
[ 80%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/SecureValue.cpp.o
[ 80%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/SendCodeHelper.cpp.o
[ 81%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/SequenceDispatcher.cpp.o
[ 81%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/SpecialStickerSetType.cpp.o
[ 81%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/StateManager.cpp.o
[ 81%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/StickersManager.cpp.o
[ 82%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/StorageManager.cpp.o
[ 82%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/SuggestedAction.cpp.o
[ 82%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/Td.cpp.o
[ 82%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/TdDb.cpp.o
[ 83%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/TermsOfService.cpp.o
[ 83%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/TopDialogManager.cpp.o
[ 83%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/UpdatesManager.cpp.o
[ 83%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/Venue.cpp.o
[ 84%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/VideoNotesManager.cpp.o
[ 84%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/VideosManager.cpp.o
[ 84%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/VoiceNotesManager.cpp.o
[ 84%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/WebPageBlock.cpp.o
[ 84%] Building CXX object CMakeFiles/tdcore.dir/td/telegram/WebPagesManager.cpp.o
[ 85%] Building CXX object CMakeFiles/tdcore.dir/td/generate/auto/td/mtproto/mtproto_api.cpp.o
[ 85%] Building CXX object CMakeFiles/tdcore.dir/td/generate/auto/td/telegram/telegram_api.cpp.o
[ 85%] Building CXX object CMakeFiles/tdcore.dir/td/generate/auto/td/telegram/secret_api.cpp.o
[ 85%] Linking CXX static library libtdcore.a
[ 85%] Built target tdcore
[ 85%] Building CXX object benchmark/CMakeFiles/bench_tddb.dir/bench_tddb.cpp.o
[ 85%] Building CXX object CMakeFiles/tdclient.dir/td/telegram/Client.cpp.o
[ 85%] Building CXX object benchmark/CMakeFiles/bench_misc.dir/bench_misc.cpp.o
[ 85%] Building CXX object CMakeFiles/tdclient.dir/td/telegram/Log.cpp.o
[ 86%] Building CXX object benchmark/CMakeFiles/bench_handshake.dir/bench_handshake.cpp.o
[ 86%] Linking CXX executable bench_handshake
[ 86%] Built target bench_handshake
[ 86%] Linking CXX executable bench_tddb
[ 86%] Built target bench_tddb
[ 86%] Linking CXX executable bench_misc
[ 86%] Built target bench_misc
[ 86%] Linking CXX static library libtdclient.a
[ 86%] Built target tdclient
[ 86%] Building CXX object benchmark/CMakeFiles/check_proxy.dir/check_proxy.cpp.o
[ 86%] Building CXX object CMakeFiles/tg_cli.dir/td/generate/auto/td/telegram/td_api_json.cpp.o
[ 86%] Building CXX object CMakeFiles/tg_cli.dir/td/telegram/cli.cpp.o
[ 86%] Building CXX object CMakeFiles/tdjson_private.dir/td/generate/auto/td/telegram/td_api_json.cpp.o
[ 86%] Building CXX object test/CMakeFiles/run_all_tests.dir/http.cpp.o
[ 87%] Building CXX object test/CMakeFiles/run_all_tests.dir/main.cpp.o
[ 89%] Building CXX object CMakeFiles/tdjson_private.dir/td/telegram/ClientJson.cpp.o
[ 89%] Building CXX object test/CMakeFiles/run_all_tests.dir/db.cpp.o
[ 89%] Building CXX object test/CMakeFiles/run_all_tests.dir/mtproto.cpp.o
[ 89%] Linking CXX executable check_proxy
[ 89%] Building CXX object test/CMakeFiles/run_all_tests.dir/message_entities.cpp.o
[ 89%] Built target check_proxy
[ 90%] Building CXX object test/CMakeFiles/run_all_tests.dir/poll.cpp.o
[ 90%] Building CXX object test/CMakeFiles/run_all_tests.dir/secret.cpp.o
[ 90%] Building CXX object test/CMakeFiles/run_all_tests.dir/secure_storage.cpp.o
[ 90%] Building CXX object test/CMakeFiles/run_all_tests.dir/set_with_position.cpp.o
[ 91%] Building CXX object test/CMakeFiles/run_all_tests.dir/string_cleaning.cpp.o
[ 91%] Building CXX object test/CMakeFiles/run_all_tests.dir/tdclient.cpp.o
[ 91%] Building CXX object test/CMakeFiles/run_all_tests.dir/tqueue.cpp.o
[ 91%] Building CXX object test/CMakeFiles/run_all_tests.dir/data.cpp.o
[ 92%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/bitmask.cpp.o
[ 92%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/buffer.cpp.o
[ 92%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/ConcurrentHashMap.cpp.o
[ 92%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/crypto.cpp.o
[ 93%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/Enumerator.cpp.o
[ 93%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/EpochBasedMemoryReclamation.cpp.o
[ 93%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/filesystem.cpp.o
[ 93%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/gzip.cpp.o
[ 94%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/HazardPointers.cpp.o
[ 94%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/heap.cpp.o
[ 94%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/json.cpp.o
[ 94%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/List.cpp.o
[ 94%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/log.cpp.o
[ 95%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/misc.cpp.o
[ 95%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/MpmcQueue.cpp.o
[ 95%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/MpmcWaiter.cpp.o
[ 95%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/MpscLinkQueue.cpp.o
[ 96%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/OptionParser.cpp.o
[ 96%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/OrderedEventsProcessor.cpp.o
[ 96%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/port.cpp.o
[ 96%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/pq.cpp.o
[ 97%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/SharedObjectPool.cpp.o
[ 97%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/SharedSlice.cpp.o
[ 97%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/StealingQueue.cpp.o
[ 97%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdutils/test/variant.cpp.o
[ 98%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdactor/test/actors_main.cpp.o
[ 98%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdactor/test/actors_simple.cpp.o
[ 98%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdactor/test/actors_workers.cpp.o
[ 98%] Building CXX object test/CMakeFiles/run_all_tests.dir/__/tdactor/test/actors_bugs.cpp.o
[ 98%] Linking CXX executable tg_cli
[ 98%] Linking CXX static library libtdjson_private.a
[ 98%] Built target tdjson_private
[ 98%] Building CXX object CMakeFiles/tdjson_static.dir/td/telegram/td_log.cpp.o
[ 98%] Building CXX object CMakeFiles/tdjson_static.dir/td/telegram/td_json_client.cpp.o
[ 98%] Building CXX object CMakeFiles/tdjson.dir/td/telegram/td_log.cpp.o
[ 98%] Building CXX object CMakeFiles/tdjson.dir/td/telegram/td_json_client.cpp.o
[100%] Linking CXX static library libtdjson_static.a
[100%] Linking CXX shared library libtdjson.dylib
[100%] Built target tdjson_static
[100%] Built target tg_cli
[100%] Built target tdjson
[100%] Linking CXX executable run_all_tests
ld: file not found: /nix/store/vf55lp3k5xgx1vlkysb7pbw76f1wh21l-clang-wrapper-11.1.0/resource-root/lib/darwin/libclang_rt.ubsan_osx_dynamic.dylib
clang-11: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [test/CMakeFiles/run_all_tests.dir/build.make:765: test/run_all_tests] Error 1
make[1]: *** [CMakeFiles/Makefile2:1156: test/CMakeFiles/run_all_tests.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

### Notify maintainers
@siraben

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"aarch64-darwin"`
 - host os: `Darwin 21.1.0, macOS 12.0`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.4pre20210922_bcd73eb`
 - channels(root): `"nixpkgs-21.11pre318361.a3a23d9599b"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute: tdlib
# a list of nixos modules affected by the problem
module:
```

--- a/pkgs/applications/networking/instant-messengers/pidgin-plugins/tdlib-purple/default.nix
+++ b/pkgs/applications/networking/instant-messengers/pidgin-plugins/tdlib-purple/default.nix
@@ -29,6 +29,8 @@ stdenv.mkDerivation rec {
 
   cmakeFlags = [ "-DNoVoip=True" ]; # libtgvoip required
 
+  NIX_CFLAGS_COMPILE = lib.optionals (stdenv.isDarwin && stdenv.isAarch64) [ "-U__ARM_NEON__" ];
+
   meta = with lib; {
     homepage = "https://github.com/ars3niy/tdlib-purple";
     description = "libpurple Telegram plugin using tdlib";

--- a/pkgs/development/libraries/tdlib/default.nix
+++ b/pkgs/development/libraries/tdlib/default.nix
@@ -23,6 +23,8 @@ stdenv.mkDerivation rec {
                  function(generate_pkgconfig' \
       --replace '\$'{prefix}/'$'{CMAKE_INSTALL_LIBDIR} '$'{CMAKE_INSTALL_FULL_LIBDIR} \
       --replace '\$'{prefix}/'$'{CMAKE_INSTALL_INCLUDEDIR} '$'{CMAKE_INSTALL_FULL_INCLUDEDIR}
+  '' + lib.optionalString (stdenv.isDarwin && stdenv.isAarch64) ''
+    sed -i "/vptr/d" test/CMakeLists.txt
   '';
 
   meta = with lib; {

